pipeline-notebooks/pipeline-general.ipynb

```diff
@@ -756,6 +757,14 @@
     "    xml_keep_tags_str = (m_xml_keep_tags[0] if len(m_xml_keep_tags) else \"false\").lower()\n",
     "    xml_keep_tags = xml_keep_tags_str == \"true\"\n",
     "    \n",
+    "    pdf_infer_table_structure = (\n",
+    "        m_pdf_infer_table_structure[0] if len(m_pdf_infer_table_structure) else \"false\"\n",
+    "    ).lower()\n",
+    "    if strategy == \"hi_res\" and pdf_infer_table_structure == \"true\":\n",
```

Comment:
Since the new model is being integrated as a strategy soon, will this parameter also need to be set in that case as well? Maybe we can store a set of all the strategies that use the value of this parameter, for future expansion.

Reply:
only hi_res for now